doc/nrf/templates/cheat_sheet.rst

```diff
@@ -403,9 +481,20 @@ To link to doxygen macros, enums or functions use:
 * :c:macro:`BT_HIDS_INFORMATION_LEN`
 * :c:func:`bt_hids_init`
```

Comment:
How about the rest? 
* Structure: ``:c:struct:``
* Type: ``:c:type:``
* Enum (the list): ``:c:enum:``
* Enumerator (an item): ``:c:enumerator:``
* Structure member: ``:c:member:``